api/auth/auth.go

```diff
@@ -200,27 +197,24 @@ func (auth *Auth) WrapHandler(h http.Handler) http.Handler {
 		auth.lock.RUnlock()
 
 		if err != nil { // Probably because signature wrong
-			w.WriteHeader(http.StatusUnauthorized)
 			// Error is intentionally dropped here as there is nothing left to
 			// do with it.
-			_, _ = io.WriteString(w, fmt.Sprintf("invalid auth token: %s", err))
+			writeUnauthorizedResponse(w, fmt.Errorf("invalid auth token: %s", err))
 			return
 		}
 		if !token.Valid { // Check that token isn't expired
-			w.WriteHeader(http.StatusUnauthorized)
 			// Error is intentionally dropped here as there is nothing left to
 			// do with it.
-			_, _ = io.WriteString(w, "invalid auth token. Is it expired?")
+			writeUnauthorizedResponse(w, errors.New("invalid auth token. Is it expired?"))
```

Comment:
Can you declare this error (and others) at the top under a `var` block?

Reply:
Initially I declared them at the top, but since [this one](https://github.com/ava-labs/avalanchego/pull/714/files#diff-26263389709bb74db688fd6e5917683aecf93bbe964d8196895e7f32f0986447R202) requires interpolation I left them as is. Any suggestion regarding this error?

Reply:
Yup, good point, I meant just the ones that don't use interpolation